include/RE/T/TESQuest.h

```diff
@@ -150,12 +151,14 @@ namespace RE
 		// members
 		std::uint64_t unk00;       // 00
 		TESCondition  conditions;  // 08
-		std::uint8_t  alias;       // 10
-		std::uint8_t  unk11;       // 11
-		std::uint16_t unk12;       // 12
-		std::uint32_t unk14;       // 14
+		std::uint32_t alias;       // 10
+		std::uint32_t unk14;       // 14 - padding?
+		// These two fields might be a structure of some sort - in ae, the function
+		// w/ id 30980 frees both of these fields, taking a pointer to the first.
+		BSTArray<BSTempEffect*> unk18;  // 18
+		BSTArray<BSTempEffect*> unk30;  // 30
 	};
-	static_assert(sizeof(TESQuestTarget) == 0x18);
+	static_assert(sizeof(TESQuestTarget) == 0x48);
```

Comment:
I only did enough re to determine that these two fields are present; there might be more after them.  I have tested added targets to objectives, though, and had no issues with that.